### ISSUE TITLE
All versions for 1.19.4 crash

### ISSUE THREAD

**Issue by 0xPascal** [SirEdvin/Turtlematic#21]
Versions 0.6.5 and 0.6.4 crashes after pressing "E".
All the versions before 0.6.4, the game crashes before opening.

Here is the crash report for version 0.6.5:
[latest.log](https://github.com/user-attachments/files/16394898/latest.log)

**Comment by SirEdvin** [comment]
Thanks for report, it seems that it is CC:T incompitablity, I would try to fix it ...

**Comment by 0xPascal** [comment]
Right, CC:T version 1.105.0 works with the mod without an issue, the versions after that has a problem with Turtlematic it seems.

**Comment by SirEdvin** [comment]
Sorry, I will not fix this as promised, I droppped 1.19.4 support.
